syntax = "proto3";

package ptmp;

service MetadataProcessor {
    rpc ProcessPostTrainingMetadata (PostTrainingMetadataRequest) returns (PostTrainingMetadataResponse) {}
}

message PostTrainingMetadataRequest {
    int32 training_id = 1;
    string data = 2;
}

message PostTrainingMetadataResponse {
}